conf.py

```diff
@@ -201,3 +203,9 @@
 # -- GraphViz configuration ----------------------------------
 
 graphviz_output_format = 'svg'
+
+
+# -- Suppress warnings  ---------------------------------------------
+# Suppress warnings for identical section titles in different documents:
+# /Users/katjasuss/Plone/trainingmasteringplone/themed_training/mastering-plone/add-ons.md:215: WARNING: duplicate label internationalization, other instance in /Users/katjasuss/Plone/trainingmasteringplone/themed_training/mastering-plone-5/add-ons.md
+suppress_warnings = ['autosectionlabel.*']
```

Comment:
This can be resolved by following the [instructions in the extension](https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html#confval-autosectionlabel_prefix_document).

Add this configuration option.
```
autosectionlabel_prefix_document=True
```

I'm not sure whether this requires changing the target markup.

Alternatively, follow the pattern of creating unique labels, as I did in this commit:
https://github.com/plone/training/pull/515/files#diff-90f1fdefddcf6bb3972af1785c216315e1d1af8e1b34b8056e686290190205e8R1

Reply:
And if we go with unique label names, do we need this extension?

Reply:
I remove sphinx.ext.autosectionlabel. Thought it could be useful but I regard it as a stupid idea of mine: section titles change and links would break.